util.py

```diff
@@ -382,86 +382,49 @@ def validate_services(services, schema_path, no_id_check, protocol_versions):
                         valid = False
                     request_params = method["request"].get("params", [])
                     method_name = service["name"] + "#" + method["name"]
-                    if not is_parameters_ordered_and_semantically_correct(
-                        method["since"], method_name + "#request", request_params, protocol_versions
+                    if not are_parameters_ordered(
+                        method["since"], method_name + "#request", request_params,
                     ):
                         valid = False
                     response_params = method["response"].get("params", [])
-                    if not is_parameters_ordered_and_semantically_correct(
+                    if not are_parameters_ordered(
                         method["since"],
                         method_name + "#response",
                         response_params,
-                        protocol_versions,
                     ):
                         valid = False
                     events = method.get("events", [])
                     for event in events:
                         event_params = event.get("params", [])
-                        if not is_parameters_ordered_and_semantically_correct(
+                        if not are_parameters_ordered(
                             event["since"],
                             method_name + "#" + event["name"] + "#event",
                             event_params,
-                            protocol_versions,
                         ):
                             valid = False
     return valid
 
 
-def is_semantically_correct_param(version, protocol_versions):
```

Comment:
Who will check the semantically correctness now? The reviewer?

Reply:
No, we don't need this anymore. Previously, we were not OK with skipping a protocol version in definitions (you couldn't use 2.8 when there were no 2.7 defined). But, after sticking to Hazelcast versions, we don't do this anymore. If there will be no changes in the 5.4 release, but some additions to the protocol in 5.5, we will now accept the usage of 5.5 in since versions